src/org/omegat/core/data/RealProject.java

```diff
@@ -1951,21 +1944,26 @@ protected String getSegmentTranslation(String id, int segmentIndex, String segme
     }
 
     static class AlignFilesCallback implements IAlignCallback {
-        AlignFilesCallback(ProjectProperties props) {
+        AlignFilesCallback(ProjectProperties props, FilterMaster filterMaster, Segmenter segmenter) {
```

Comment:
The order of argument.
`ExternalTMFactory.load` takes `segmenter` then `filterMaster` but `AlignFilesCallback` ctor takes `filtermaster` then `segmenter`.

Reply:
As an alternative, is it really relevant to keep this class static?